app/controllers/ingredient_families_controller.rb

```diff
@@ -0,0 +1,9 @@
+class IngredientFamiliesController < ApplicationController
+  def index
+    @ingredient_families = IngredientFamily.all
```

Comment:
When we ask `Something.all` - postgresql gives out data in a strange order. It's always better to explicitly ask for an order. Like that: `IngredientFamily.default_scope` and then add `default_scope { order(:id) }` to the model. This way we ensure that ingredient families we get will always be ordered by id.